shuttle/templates/schedules/request_shuttle.html

```diff
@@ -11,31 +11,61 @@ <h4>Select a location and time below to request the shuttle:</h4>
     <div class="center-page">
         <div class="align-side-by-side">
             <div class="dropdown">
-                <button class="btn btn-primary btn-lg dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Location</button>
+                <button class="btn btn-primary btn-lg dropdown-toggle location-menu" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Location</button>
                 <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-                    <a class="dropdown-item" href="#">*Insert location here*</a>
-                    <a class="dropdown-item" href="#">*Insert more locations*</a>
-                </div>
-            </div>
-        </div>
-        <div class="align-side-by-side">
-            <div class="dropdown">
-                <button class="btn btn-primary btn-lg dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Time</button>
-                <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-                   <a class="dropdown-item" href="#">*Insert time here*</a>
-                    <a class="dropdown-item" href="#">*Insert more times here*</a>
+                    {% for location in locations %}
+                        <a class="dropdown-item" href="#">{{ location }}</a>
+                    {% endfor %}
                 </div>
             </div>
         </div>
         <div class="align-side-by-side">
             <div class="submit">
-                <input class="btn btn-warning btn-lg" type="submit" value="Request Shuttle">
+                <input class="btn btn-warning btn-lg submit-btn" type="submit" value="Request Shuttle">
             </div>
         </div>
     </div>
 
     <div class="center-page">
-       <h4>Students awaiting shuttle: <span class="badge badge-light">*Insert number here*</span></h4>
+       <h4>Current Shuttle Waitlist: <span class="badge badge-pill badge-warning"></span></h4>
```

Comment:
Maybe swap `Current Shuttle Waitlist` and the selector stuff so that the waitlist is the most prominent information on that page. Also add some space between the waitlist and selector after you move it.

Reply:
Alright I made those changes